gulpfile.js

```diff
@@ -75,6 +75,28 @@ if (argv.enable_sendmail) {
   params.enable_sendmail = argv.enable_sendmail;
 }
 
+// Check if path to the file to be minified and/or concatenated does exist.
+// If not, raise a warning and terminate the program.
+// This will help toc heck spelling errors in manifest.json.
+var checkPath = {
```

Comment:
Not sure why these need to be combined in an object, they seem more like separate utility functions to me. You could do:

  requireFileExists(filePath)
  requireDirExists(dirPath)

which does nothing if the file/dir exists, and throws an error (and terminates) otherwise.